src/gui-wizard-gtk/wizard.c

```diff
@@ -2101,8 +2101,8 @@ static GList *find_words_in_text_buffer(int page,
 
             if (g_list_find_custom(ignore_sitem_list, found_word, (GCompareFunc)sitem_contains))
             {
-                if (found_word)
-                    free(found_word);
+                g_free(found_word);
+                found_word = NULL;
```

Comment:
This would be even shorter using `g_clear_pointer()`.

Reply:
True. I will update this pattern in the other PRs as well.